micrometer-tracing-bridges/micrometer-tracing-bridge-otel/src/main/java/io/micrometer/tracing/otel/bridge/DefaultHttpClientAttributesGetter.java

```diff
@@ -38,6 +39,30 @@ public String getUrlFull(HttpClientRequest httpClientRequest) {
         return httpClientRequest.url();
     }
 
+    @Nullable
+    @Override
+    public String getServerAddress(HttpClientRequest httpClientRequest) {
```

Comment:
Can we migrate to the new version of OTel separately?

Reply:
Done